scripts/rollup/build.js

```diff
@@ -78,6 +78,7 @@ function getHeaderSanityCheck(bundleType, hasteName) {
 }
 
 function getBanner(bundleType, hasteName, filename) {
+  const isReconciler = /react-reconciler/.test(filename);
```

Comment:
I would prefer a flag on the bundle, like we do for `isRenderer`. Or, ideally, unify them under a required enum: `'isomorphic' | 'renderer' | 'reconciler'`. But unification can wait.